test/ng/compileSpec.js

```diff
@@ -7430,3 +7431,19 @@ describe('$compile', function() {
       }));
   });
 });
+
+describe('$compile', function() {
+  describe('directive', function() {
+    it('should fail to register a directive', function() {
+      try {
+        module(provideLog, function($provide, $compileProvider) {
```

Comment:
I don't need `provideLog` in particular - is there a default noop module I can instantiate here, or should I create a new one?